gpii/node_modules/flowManager/configs/gpii.flowManager.config.production.base.json5

```diff
@@ -0,0 +1,12 @@
+// This config turns on the local flowManager flag "resetAtStart" to reset the system to default settings on system startup.
+{
+    "type": "gpii.flowManager.config.production.base",
```

Comment:
This wasn't quite what I meant - I didn't mean to add fresh derived versions of *all* of our configs - I meant to just add derived versions of the *one* config which we intend Astea to use in production for the actual deployment of the app to the pilots etc., which I believe is gpii.config.untrusted.development.dynamicDR.all.local . The derived version we should then just call gpii.config.production.local
We have far too many configs as it is and adding a fresh layer of derivation to all of them will just confuse people further (including Astea!)